frontend/src/metabase/admin/settings/components/SettingsAuthenticationForm.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router'
+
+class SettingsAuthenticationForm extends Component {
```

Comment:
Naming nitpicking: could this be something like `SettingsAuthenticationOptions`?